experimental/lib/Support/FtdSupport.cpp

```diff
@@ -104,22 +171,21 @@ BoolExpression *experimental::ftd::getPathExpression(
 
     if (ignoreDeps || deps.contains(firstBlock)) {
 
-      // Get last operation of the block, also called "terminator"
+      // Get last operation of the block, also called `terminator`
       Operation *terminatorOp = firstBlock->getTerminator();
 
       if (isa<cf::CondBranchOp>(terminatorOp)) {
-        unsigned blockIndex = mapBlockToIndex.lookup(firstBlock);
-        std::string blockCondition = "c" + std::to_string(blockIndex);
+        unsigned blockIndex = bi.getIndexFromBlock(firstBlock);
+        std::string blockCondition = bi.getBlockCondition(firstBlock);
```

Comment:
The current GSA library still has in the code the old mechanism I used to give a index to the blocks. I'll get rid of it with the next PR.

Reply:
Sounds good, thanks for the ping.